kolibri/plugins/facility_management/assets/src/views/user-page/edit-user-modal.vue

```diff
@@ -30,7 +30,7 @@
         ref="username"
         type="text"
         :label="$tr('username')"
-        :maxlength="30"
+        :maxlength="$tr('userMaxLength', { userMaxLength: 30 })"
```

Comment:
Is this used as an an HTML attribute also? `<input type="text" maxlength="123">` ? Because in that case, translating it is a bad idea, it would have to be separated into two different properties...

Reply:
Good catch, yes. `maxlength` is a programmatic attribute, not a user-facing string.